### PR TITLE
Voice + Queue Updates

### PR DESCRIPTION
**Pull request by andrewpage**
I have implemented the following changes:

- Support the [Queue Create API](https://www.twilio.com/docs/voice/api/queue-resource#create-a-queue-resource) via `gotwilio.CreateQueue(string) (*QueueResponse, *Exception, error)`
- Support [Voice Call Update API](https://www.twilio.com/docs/voice/api/call#update-a-call-resource) via `CallUpdate(string, url.Values) (*VoiceResponse, *Exception, error)`

### CODE REVIEW

**Comment by andrewpage** [comment]
@sfreiberg ?

File: voice.go [hunk 1]
```diff
@@ -199,28 +201,37 @@ func (twilio *Twilio) CallUpdate(callSid string, formValues url.Values) (*VoiceR
...
+	body := new(bytes.Buffer)
+	_, err = body.ReadFrom(res.Body)
+	if err != nil {
+		return voiceResponse, exception, err
+	}
+
+	responseStr := body.String()
 
-	if res.StatusCode != http.StatusCreated {
+	if res.StatusCode != http.StatusOK && res.StatusCode != http.StatusAccepted && res.StatusCode != http.StatusCreated {
```
[the remark this answers is not in the document]
Comment (andrewpage) [reply voice.go]: Twilio returns a `200 OK` on successful update, which is why that change was needed for this particular change. Twilio only returns `201 CREATED` on successful creation (of a call, for instance).

Now, I don't think `202 ACCEPTED` is necessary. Not sure what my logic was there, but I think I may have added it in the interest of being as permissible as possible, but that's unnecessary. I've backed out of that change.

https://www.twilio.com/docs/usage/your-request-to-twilio#post-responses

**Comment by andrewpage** [comment]
@sfreiberg I've backed out of a number of the code style changes I had made, and addressed the question about the acceptable HTTP response codes on our POST request. Let me know if there's anything else you'd like to see changed!

**Comment by sfreiberg** [comment]
@andrewpage Thanks for the updates and the PR! It's been merged.

**Pull request merged**